README.md

```diff
@@ -0,0 +1,26 @@
+/// # Vector with holes
+/// 
+/// Vector with holes data structure. Each time an element is removed its index is added into a
+/// list of empty elements. No other values are moved or touched. This allows removing elements in
+/// a constant time.
```

Comment:
I might also say something like:
"And allows the indices of the other elements to remain stable."

Reply:
Thanks Luke. Fixed in https://github.com/vsbogd/holeyvec/pull/1/commits/70d003ed6ba15a1f31bbebe48beca0f83080dd67